stripe/src/main/java/com/stripe/android/view/CardInputView.java

```diff
@@ -69,6 +71,31 @@ public CardInputView(Context context, AttributeSet attrs, int defStyleAttr) {
         initView(attrs);
     }
 
+    /**
+     * Gets a {@link Card} object from the user input, if all fields are valid. If not, returns
+     * {@code null}.
+     *
+     * @return a valid {@link Card} object based on user input, or {@code null} if any field is
+     * invalid
+     */
+    @Nullable
+    public Card getCard() {
+        String cardNumber = mCardNumberEditText.getCardNumber();
```

Comment:
Thinking out loud (non blocking) :Would it make sense for `cardNumberEditText` , `expiryDateEditText` and `cvcNumberEditText` to have an `isValid`/`isComplete` accessor such that all validation is part of those respective fields and the consumers of this field don;t need to re-implement/implement the validation themselves?